vendor/python/astexport.py

```diff
@@ -99,6 +99,9 @@ def visit_str(self, val):
     def visit_field_NameConstant_value(self, val):
         return str(val)
 
+    def visit_Bytes(self, val):
```

Comment:
Would it be possible to upstream this? In any case we should make sure to explicitly call out what we've changed in copied libraries so they don't get accidentally removed when pulling in new versions.

Reply:
You're right! I'll do this soon :)